### ISSUE TITLE
eiwd fails to build

### ISSUE THREAD

**Issue by arnolddinkla** [kisslinux/repo#130]
Terminal output below

```
Generated manual page src/iwd.8 does not exist
make[1]: *** [Makefile:3078: src/iwd.8] Error 1
make: *** [Makefile:1220: all] Error 2
!> eiwd Build failed
```

**Comment by dylanaraps** [comment]
I can't reproduce this at all. Is this error from `kiss build eiwd`?

**Comment by arnolddinkla** [comment]
Correct. Running from a fully updated system.

**Comment by arnolddinkla** [comment]
On another pc with kiss installed. Build fails as follows

`kiss b eiwd
-> Resolving dependencies
-> Building: eiwd
-> Checking to see if any dependencies have already been built
-> Installing any pre-built dependencies
-> eiwd Checking repository files
-> eiwd Downloading sources
-> eiwd Found cached source 'eiwd-0.2.1.tar.xz'
-> eiwd Extracting sources
-> eiwd Starting build
configure: error: cannot find install-sh, install.sh, or shtool in build-aux "."/build-aux
!> eiwd Build failed
`

**Comment by dylanaraps** [comment]
I've pushed a new version of `eiwd` which should fix the issue. (0.2.2). `kiss u` and then `kiss b eiwd` will build the new version.

**Comment by dylanaraps** [comment]
OK. This is fixed in `eiwd` `0.2.3` as I was able to reproduce and finally fix the issue. Thanks for your patience.
